src/sentry/snuba/tasks.py

```diff
@@ -250,14 +252,23 @@ def _create_snql_in_snuba(subscription, snuba_query, snql_query, entity_subscrip
         )
 
     entity_key = get_entity_key_from_request(snql_query)
+
+    if in_random_rollout("snuba.snql.enable-orjson"):
+        post_body = orjson.dumps(body).decode("utf-8")
```

Comment:
`.decode()` is slightly faster than `.decode('utf-8')` and the default is utf-8 in python 3

Reply:
Nice catch. You're absolutely right.